c2corg_ui/static/js/appmodule.js

```diff
@@ -26,5 +26,6 @@ app.module = angular.module('app', [
   'ngFileUpload',
   'slug',
   'vcRecaptcha',
-  'infinite-scroll'
+  'infinite-scroll',
+  'ngSanitize'
```

Comment:
This is need to insert HTML using ng-bind-html. I have a "no safe content ..." error without ngSanitize.
https://docs.angularjs.org/api/ng/directive/ngBindHtml